nais/dev-fss.yml

```diff
@@ -46,9 +46,6 @@ spec:
     requests:
       cpu: 200m
       memory: 512Mi
-  ingresses:
```

Comment:
Du trenger nok disse ingressene om det skal fungere i fss :)

Reply:
usikker på hvorfor jeg har vært her i det hele tatt, for tanken er jo at denne skal slettes :p